metabase/metabase
# Fix UX for empty dashboard filter with multiple choices

Closes https://github.com/metabase/metabase/issues/25533

### Description

Solution is pretty similar to what we have in other places, as it's more UX problem and not code problem, we need to explicitly show to use that no data is selected in the filter to avoid confusing.
We should be able to update filter even with empty values to handle a case with unchecked checkboxes in the filter

### How to verify

- open dashboard with a filter (choose category and multiple options to select)
- choose some value in the filter, save
- then unselect selected values, save
- parameter filter should be in the initial state (not like before - the empty value and cross icon)

### Demo

https://www.loom.com/share/1551740d8e704571b203da78843cd0f6

### Checklist
